quick-start/kratos/kratos.yml

```diff
@@ -42,12 +42,12 @@ selfservice:
         default_browser_return_url: /
 
     login:
-      ui_url: /uc/auth/login
+      ui_url: http://one.erda.local/uc/auth/login
       lifespan: 10m
 
     registration:
       lifespan: 10m
-      ui_url: /uc/auth/registration
+      ui_url: http://one.erda.local/uc/auth/registration 
```

Comment:
add comment in code. Not just on conversation.

Reply:
done